src/OpenColorIO/Config.cpp

```diff
@@ -5064,11 +4727,112 @@ void Config::serialize(std::ostream& os) const
     }
 }
 
-void Config::setProcessorCacheFlags(ProcessorCacheFlags flags) noexcept
+ProcessorCacheFlags Config::getProcessorCacheFlags() const noexcept
+{
+    return getImpl()->getProcessorCacheFlags();
+}
+
+void Config::setProcessorCacheFlags(ProcessorCacheFlags flags) const noexcept
 {
     getImpl()->setProcessorCacheFlags(flags);
 }
 
+//////////////////////////////////////////////////////////////////
```

Comment:
Lines 4740-4835 is simply a cut/paste move from farther down in this module.  This code is from another one of our recent PRs and we decided it belongs better in this location.